language/vm/vm-runtime/src/move_vm.rs

```diff
@@ -88,6 +88,20 @@ impl MoveVM {
             .rent(|runtime| runtime.publish_module(module, chain_state, txn_data))
     }
 
+    pub fn cache_module(&mut self, module: VerifiedModule) {
```

Comment:
those 2 functions are needed now and truth is the may stay around forever, not too clear and we'll see in time